packages/wrangler/src/secret/index.ts

```diff
@@ -563,3 +538,47 @@ export function validateJSONFileSecrets(
 		}
 	}
 }
+
+export async function parseBulkInputToObject(input?: string) {
+	let content: Record<string, string>;
+	if (input) {
+		const jsonFilePath = path.resolve(input);
+		try {
+			const fileContent = readFileSync(jsonFilePath);
+			try {
+				content = parseJSON<Record<string, string>>(fileContent);
+			} catch (e) {
+				content = dotenvParse(fileContent);
+				// dotenvParse does not error unless fileContent is undefined, no keys === error
```

Comment:
What if it's not a valid env file format? Something like:

```
2:4
3==5
```

Reply:
```javascript
import { parse } from "dotenv";

console.log(
	parse(`2:4
4==5`)
);
//  { '4': '5' }

```

dotenv parse is rather lax, it only throws an exception if you pass undefined which is guarded against. in the case it can't parse the file it just returns an empty object